components/ModEditFacilityLeftNavBar.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+    <div class="flex flex-col">
+        <h1 class="font-bold text-lg p-1 mb-2 text-ellipsis">
+            {{ moderationSubmissionsStore.selectedSubmissionData?.facility?.nameEn
+                || $t("modPanelFacilityLeftNavbar.facilityNameUnknown") }}
+        </h1>
+        <div class="flex flex-col items-start">
+            <button
+                data-testid="submission-form-leftnav-contact-information"
+                :class="{ 'bg-secondary': activeSection === ModEditFacilityLeftBarSectionIDs.ContactInformation,
+                          'bg-primary-inverted': activeSection !== ModEditFacilityLeftBarSectionIDs.ContactInformation }"
+                class="w-full py-4 my-2 text-sm text-start pl-2 rounded border-b-2 border-tertiary-bg"
```

Comment:
Since these are all repeated, we could use tailwinds `@apply` here to simplify our repeated styling 🚀 

Example
![image](https://github.com/user-attachments/assets/6963b7bd-700e-4e3a-be09-a7600e7f99bb)